src/pages/TransactionInfoPage.tsx

```diff
@@ -115,138 +115,131 @@ const TransactionInfoPage = () => {
-                    </td>
-                  </TableRow>
-                )}
-                {isTxConfirmed(txInfo) && (
-                  <TableRow>
-                    <td>Inputs</td>
-                    <td>
-                      {txInfo.inputs && txInfo.inputs.length > 0
-                        ? txInfo.inputs.map((v, i) => (
-                            <AddressLink
-                              address={v.address}
-                              txHashRef={v.txHashRef}
-                              key={i}
-                              amount={BigInt(v.amount)}
-                            />
-                          ))
-                        : 'Block Rewards'}
-                    </td>
-                  </TableRow>
-                )}
-                <TableRow>
-                  <td>Outputs</td>
-                  <td>
-                    {isTxConfirmed(txInfo) &&
-                      txInfo.outputs &&
-                      txInfo.outputs.map((v, i) => (
-                        <AddressLink address={v.address} key={i} amount={BigInt(v.amount)} txHashRef={v.spent} />
-                      ))}
+                    </span>
                   </td>
                 </TableRow>
+              )}
+              {isTxConfirmed(txInfo) && txInfo.timestamp && (
                 <TableRow>
-                  <td>Gas Amount</td>
-                  <td>{txInfo.gasAmount || '-'} GAS</td>
-                </TableRow>
-                <TableRow>
-                  <td>Gas Price</td>
+                  <td>Timestamp</td>
                   <td>
-                    <Amount value={BigInt(txInfo.gasPrice)} showFullPrecision />
+                    <Timestamp timeInMs={txInfo.timestamp} forceHighPrecision />
                   </td>
                 </TableRow>
+              )}
+              {isTxConfirmed(txInfo) && (
                 <TableRow>
-                  <td>Transaction Fee</td>
+                  <td>Inputs</td>
                   <td>
-                    <Amount value={BigInt(txInfo.gasPrice) * BigInt(txInfo.gasAmount)} showFullPrecision />
+                    {txInfo.inputs && txInfo.inputs.length > 0
+                      ? txInfo.inputs.map((v, i) => (
+                          <AddressLink address={v.address} txHashRef={v.txHashRef} key={i} amount={BigInt(v.amount)} />
+                        ))
+                      : 'Block Rewards'}
                   </td>
                 </TableRow>
+              )}
+              {isTxConfirmed(txInfo) && (
                 <TableRow>
+                  <td>Outputs</td>
                   <td>
-                    <b>Total Value</b>
-                  </td>
-                  <td>
-                    <Badge
-                      type="neutralHighlight"
-                      amount={outputs && outputs.reduce<bigint>((acc, o) => acc + BigInt(o.amount), BigInt(0))}
-                    />
+                    {txInfo.outputs
```

Comment:
* It should be impossible for outputs to be absent from transaction information (nature of UTXO)
* `[] ? 'truthy' : 'falsey' === 'truthy'` will evaluate to `true`, so if `outputs` is an empty list nothing will appear - it'd probably be helpful to show "no outputs" instead but this situation is impossible in the first place because of the first point